src/fhir/1_Condition_DCW_MP_1.json

```diff
@@ -0,0 +1,39 @@
+{
+  "resourceType": "Condition",
+  "id": "condition-DCW-MP-1",
+  "clinicalStatus": {
+    "coding": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/condition-clinical",
+        "code": "active",
+        "display": "Active"
+      }
+    ],
+    "text": "Active"
+  },
+  "verificationStatus": {
+    "coding": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+        "code": "confirmed",
+        "display": "Confirmed"
+      }
+    ],
+    "text": "Confirmed"
+  },
+  "code": {
+    "coding": [
+      {
+        "system": "http://www.cdc.gov/nchs/icd/icd10cm.htm",
+        "code": "F33.1",
+        "display": "Major Depressive Disorder, Recurrent, Moderate"
+      }
+    ],
+    "text": "Major Depressive Disorder, Recurrent, Moderate"
+  },
+  "subject": {
+    "reference": "Patient/patient-dcw-mp"
```

Comment:
same as my previous comment, not sure if this is referencing subject `patient-dcw-ms` or `patient-dcw-pm` as I don't see a patient with an ID of `patient-dcw-mp`

Reply:
Nice find @achen2401  - I think I've fixed it now.